### ISSUE TITLE
/css-variables-1_dev/html/variable-font-face-01.htm intermittently fails to load ahem

### ISSUE THREAD

**Issue by Ms2ger** [servo/servo#7870]

**Comment by mbrubeck** [comment]
```
/css-variables-1_dev/html/variable-external-font-face-01.htm
------------------------------------------------------------
FAIL [Parent]
```

**Comment by eefriedman** [comment]
I think the problem here isn't that the font fails to load... it just fails to load in time for the reftest screenshot.  When I look at the test using just "./mach run", I can see it initially paints with the wrong font, then repaints with the correct font.

In addition to making reftests fail, this seems bad for performance: we're doing extra, useless reflows.

**Comment by mbrubeck** [comment]
Possibly related to #7596. cc @pcwalton

**Comment by eefriedman** [comment]
See #8257; I think that's the immediate problem, although I'm not completely sure.

**Comment by mbrubeck** [comment]
Appears fixed by #8257.
